example/flutter/pubspec.yaml

```diff
@@ -0,0 +1,6 @@
+name: usage_example
+dependencies:
+  usage:
+    path: ../..
+  flutter:
+    path: ../../../flutter/packages/flutter
```

Comment:
I'm not sure what the alternative is? Change to "/path/to/flutter/packages/flutter" and make them edit it manually?